src/routes/safe/components/Transactions/TxList/TxDataRow.tsx

```diff
@@ -21,29 +22,42 @@ type TxDataRowType = {
   hasExplorer?: boolean
   title: string
   value?: string
+  isArray?: boolean
+  method?: string
+  paramType?: string
 }
 
-export const TxDataRow = ({ children, inlineType, hasExplorer = true, title, value }: TxDataRowType): ReactElement => (
+export const TxDataRow = ({
+  children,
+  inlineType,
+  hasExplorer = true,
+  title,
+  value,
+  isArray,
+  method,
+  paramType,
+}: TxDataRowType): ReactElement => (
   <StyledGridRow>
-    <Text size="xl" as="span">
+    <Text size="xl" as="span" color="placeHolder">
       {title}
     </Text>
-    {value && inlineType === 'hash' && (
-      <InlineEthHashInfo
+    {isArray && value && method && paramType && <Value method={method} type={paramType} value={value as string} />}
```

Comment:
Do you need cast here? You check that `value exists`.

Reply:
It was a leftover 👍